.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.4.3-node
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - middleman-template-{{ checksum "Gemfile.lock" }}
+
+      - run:
+          name: Install Ruby dependencies
+          command: bundle install --path vendor/bundle
+
+      - save_cache:
+          key: middleman-template-{{ checksum "Gemfile.lock" }}
```

Comment:
They recommend keying from the architecture that circle happens to use for the run. I've never seen it anything other than some linux machine, but it doesn't hurt.

something like:
`key: middleman-template-{{ arch }}-{{ checksum "Gemfile.lock" }}`

[their example](https://circleci.com/docs/2.0/caching/#full-example-of-saving-and-restoring-cache)

Reply:
Oh, nice! Do you think we should also use the `{{ .Branch }}` key, and/or use multiple caches, as they do in that example? Or is that overkill in this case?

Reply:
I figure if the `Gemfile.lock` is the same across branches, it'd be ok to share the cache, so I suspect it's overkill.